src/components/Counter.js

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {CounterButton} from "./CounterButton";
+
+export class Counter extends React.PureComponent {
+    constructor(props) {
+        super(props);
+        this.state = {
+            count: 1
+        };
+    }
+
+    updateCounter = (e) => {
```

Comment:
nit: it's better to have two separate handlers for increment and decrement.
This function is responsible for two things, hence violates single responsibility principle